lib/improver/utilities/cube_manipulation.py

```diff
@@ -249,6 +261,13 @@ def process(self, cubes_in):
             ValueError:
                 If master coordinate is not present on all "cubes_in"
         """
+        # if input is a single cube or list of length 1, return unchanged
+        if isinstance(cubes_in, iris.cube.Cube):
```

Comment:
Do we have tests for these to if statements?

Reply:
Added